README.md

```diff
@@ -1 +1,46 @@
-# ToDo-Review
\ No newline at end of file
+# Today's To Do
+
+## This is a simple daily todo application
+
+## Getting Started
+
+1. copy the Url from my github account
+2. clone the below url to your local machine
+3. Run git clone git@github.com:ismailakinkunmi/portfolio-site.git
+
+## Technologies Used
+
+1. Eslint
+2. Webpack
+3. HTML5
+4. CSS#
+5. JavaScript
+
```

Comment:
From my observation, I have seen that these two lines are repeated two times and it is not the best coding practice to repeat things. I kindly recommend that you may remove one copy and remain with another one. In this way, you don't repeat yourself(DRY) 😃